tox.ini

```diff
@@ -97,4 +97,12 @@ commands=
     mkdir -p {toxinidir}/tmp/data
     screen -dmS {envname} python -m candig_dataset_service \
     --host {env:DATASETS_SERVICE_HOST} --port {env:DATASETS_SERVICE_PORT} \
-    --database {toxinidir}/tmp/data/datasets.db --logfile {toxinidir}/tmp/log/datasets.log)
\ No newline at end of file
+    --database {toxinidir}/tmp/data/datasets.db --logfile {toxinidir}/tmp/log/datasets.log)
+
+[testenv:authorization]
+basepython=python3.6
```

Comment:
Should we just downgrade python and pin 3.6.10 as global or continue with python3.7?